### PR TITLE
Add instructions to check weave-kube logs

### PR DESCRIPTION
**Pull request by bboreham**
"Check it is running" and "check the logs" are important steps when troubleshooting.

### CODE REVIEW

**Comment by dholbach** [comment]
Nice one!

File: site/kubernetes/kube-addon.md [hunk 1]
```diff
@@ -186,12 +186,42 @@ definition](https://v1-7.docs.kubernetes.io/docs/api-reference/v1.7/#networkpoli
...
+logs. Pick one of the pods with that status from the list and ask for
+the logs like this:
+
+```
+$ kubectl logs -n kube-system weave-net-1jkl6 weave
+```
+
+You can pipe the output into a file for easier viewing, especially if
+it is long.
+
```
Comment (abuehrle) [inline site/kubernetes/kube-addon.md]: >For easier viewing, pipe the output into a file , especially if
it is long.

File: site/kubernetes/kube-addon.md [hunk 2]
```diff
@@ -186,12 +186,42 @@ definition](https://v1-7.docs.kubernetes.io/docs/api-reference/v1.7/#networkpoli
 
 ## <a name="troubleshooting"></a> Troubleshooting
 
+The first thing to check is whether the Weave Net software is up and
+running. The `kubectl apply` command you use to install only
+_requests_ that it be downloaded and started; if anything goes wrong
+at startup the details will only be visible in the logs of the container(s).
+
```
Comment (abuehrle) [inline site/kubernetes/kube-addon.md]: The first thing to check is whether Weave Net is up and
running. The `kubectl apply` command you used to install it only
_requests_ that it be downloaded and started; if anything goes wrong
at startup, those details will only be visible in the logs of the container(s).

Suggest: include a link on how to get the logs of a container?

Comment (bboreham) [reply site/kubernetes/kube-addon.md]: The `kubectl logs` command 20 lines down is how.

Comment (abuehrle) [reply site/kubernetes/kube-addon.md]: Probably a good idea to link to it with an anchor tag.

Comment (abuehrle) [reply site/kubernetes/kube-addon.md]: I missed that. Looks good!

**Comment by bboreham** [comment]
> Can we see those in Scope or Weave Cloud?

Yes, there is a "logs" button, but it's not a great user experience if there are a lot of lines to scroll.

More importantly if your pod network is not working it is unlikely you will have successfully installed the Weave Cloud agents.

**Pull request merged**